utils/rust.py

```diff
@@ -16,9 +16,11 @@
 # DEALINGS IN THE SOFTWARE.
 
 import ctypes
+from os import path
 from typing import Tuple
 
-LIB_PATH = "/root/zkp-subnet/prover"
+here = path.abspath(path.dirname(__file__))
+LIB_PATH = path.join(here, "prover")
```

Comment:
This seems to put the path into the utils folder but we want the repo root, so could you update it to go up one folder?